### PR TITLE
Simple anti bot detection

### PR DESCRIPTION
**Pull request by kdzwinel**
Prevents detection techniques listed in https://privacy-test-pages.glitch.me/crawler/

### CODE REVIEW

File: collectors/APICalls/breakpoints.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ const breakpoints = [
             {name: 'storage'},
             {name: 'vendor'},
             {name: 'vendorSub'},
+            {name: 'webdriver'},
```
Comment (kdzwinel) [inline collectors/APICalls/breakpoints.js:131]: We can now check if property was touched by scripts w/o revealing that it exists.

Comment (sammacbeth) [reply collectors/APICalls/breakpoints.js:131]: So this will be included in the tracker-radar API call data?

**Review by sammacbeth — APPROVED** [review]
Looks great, thanks for implementing this!

File: collectors/APICalls/breakpoints.js (hunk 1 above)
Comment (kdzwinel) [reply collectors/APICalls/breakpoints.js:131]: Yep, everything we add here to the breakpoints ends up, after crawl and processing, in tracker-radar repo.

**Pull request merged**